upgrades/install.ps1

```diff
@@ -0,0 +1,11 @@
+# set environment variables
+$env:AS_VERSION_SHORT = 'AS411'
+Get-ChildItem "$PWD" -Recurse -Filter *.exe | Foreach-Object { 
+    Write-Output "installing $($_.Name)"; 
+    Write-Verbose "Into C:\BrAutomation\$env:AS_VERSION_SHORT"; 
+    & $_.FullName -G='C:\BrAutomation' -V="C:\BrAutomation\$env:AS_VERSION_SHORT" -R | Out-Null; 
+    if($LASTEXITCODE -ne 0) { 
+        Write-Error "Error installing $($_.Name). Exit code $LASTEXITCODE"; 
+        exit 1; 
+    } 
+}
```

Comment:
It looks like you're now using AsPython for upgrades, is this Powershell script still needed?

Reply:
It is not needed but I like the script. It can be used by people who are new to the project that want to install the upgrades